config/webhook/manifests.yaml

```diff
@@ -12,4 +12,61 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  creationTimestamp: null
+  name: mutating-webhook-configuration
+webhooks:
+- admissionReviewVersions:
+  - v1
+  - v1beta1
+  clientConfig:
+    service:
+      name: webhook-service
+      namespace: default
+      path: /mutate-crdb-cockroachlabs-com-v1alpha1-crdbcluster
+  failurePolicy: Fail
+  name: mcrdbcluster.kb.io
+  rules:
+  - apiGroups:
+    - crdb.cockroachlabs.com
+    apiVersions:
+    - v1alpha1
+    operations:
+    - CREATE
+    - UPDATE
+    resources:
+    - crdbclusters
+  sideEffects: None
 
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  creationTimestamp: null
+  name: validating-webhook-configuration
+webhooks:
+- admissionReviewVersions:
+  - v1
+  - v1beta1
+  clientConfig:
+    service:
+      name: webhook-service
+      namespace: default
+      path: /validate-crdb-cockroachlabs-com-v1alpha1-crdbcluster
+  failurePolicy: Fail
+  name: vcrdbcluster.kb.io
+  rules:
+  - apiGroups:
+    - crdb.cockroachlabs.com
+    apiVersions:
+    - v1alpha1
+    operations:
+    - CREATE
+    - UPDATE
+    - DELETE
```

Comment:
Do we need it on delete?

Reply:
Currently, not at all. I left this here so that if anyone adds code to `ValidateDelete`, it would indeed run. However, I think we can remove it for now and add it in later if needed 👍